softlayer/vm/vm_utils.go

```diff
@@ -43,7 +43,8 @@ func CreateVirtualGuestTemplate(agentID string, stemcell bslcstem.Stemcell, clou
 	base64EncodedMetadata := Base64EncodeData(string(metadataBytes))
 
 	virtualGuestTemplate := sldatatypes.SoftLayer_Virtual_Guest_Template{
-		Hostname:  cloudProps.VmNamePrefix + TimeStampForTime(time.Now().UTC()),
+		//Hostname:  cloudProps.VmNamePrefix + TimeStampForTime(time.Now().UTC()),
```

Comment:
If we no longer need the Hostname in the template then get rid of it here. Leaving it as a comment creates more confusion then help.